x/evm/keeper/migrate.go

```diff
@@ -30,6 +31,10 @@ func GetMigrationHandler(k BaseKeeper, n types.Nexus) func(ctx sdk.Context) erro
 			}
 			ck := k.ForChain(chain.Name).(chainKeeper)
 
+			if err := setVotingGracePeriod(ctx, ck); err != nil {
+				return err
+			}
+
 			if err := migrateContractsBytecode(ctx, ck); err != nil {
 				return err
 			}
```

Comment:
You shouldn't put the "ABSOLUTELY NEVER EVER DELETE" migration between deletable migrations